src/util/options.rs

```diff
@@ -25,7 +25,7 @@ pub enum NurseryZeroingOptions {
 }
 
 /// Select a GC plan for MMTk.
-#[derive(Copy, Clone, EnumString, Debug)]
+#[derive(Copy, Clone, EnumString, Debug, PartialEq)]
```

Comment:
It's OK for PlanSelector to implement `Eq`, too. `PartialEq` is designed for things like `f32` and `f64` where not all values are comparable (such as NaN).  `PlanSelector` is fully comparable.

Reply:
Yes. I added `Eq`.